fc-exploitation/src/service-provider/dto/service-provider-input.dto.spec.ts

```diff
@@ -46,4 +46,21 @@ describe('Service Provider Input (Data Transfer Object)', () => {
     // Then | Assert
     expect(result).toEqual([]);
   });
+
+  it('should validate if service provider user info algorithme is ""', async () => {
+    const customServiceProvider = {
+      ...serviceProvider,
+      userinfo_signed_response_alg: '',
+    };
+
+    // When | Action
+    const serviceProviderToClass = plainToClass(
+      ServiceProviderDto,
+      customServiceProvider,
+    );
+    const result = await validate(serviceProviderToClass);
+
+    // Then | Assert
+    expect(result).toEqual([]);
```

Comment:
nit-pick: this looks like you are testing class-validator here.

Reply:
Good challenge. I used the same pattern as the previous tests though. Do you recommand I drop this test altogether, or rather I change the indentation so that it's clearer that the DTO is the one on trial here?